docs/data/joy/getting-started/templates/email/components/WriteEmail.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react';
+import Box from '@mui/joy/Box';
+import ModalClose from '@mui/joy/ModalClose';
+import Button from '@mui/joy/Button';
+import FormControl from '@mui/joy/FormControl';
+import Textarea from '@mui/joy/Textarea';
+import Sheet from '@mui/joy/Sheet';
+import { IconButton, Input, Stack, Typography } from '@mui/joy';
+
+import FormatColorTextRoundedIcon from '@mui/icons-material/FormatColorTextRounded';
+import AttachFileRoundedIcon from '@mui/icons-material/AttachFileRounded';
+import InsertPhotoRoundedIcon from '@mui/icons-material/InsertPhotoRounded';
+import FormatListBulletedRoundedIcon from '@mui/icons-material/FormatListBulletedRounded';
+
+interface WriteEmailProps {
+  open?: boolean;
+  onClose?: () => void;
+}
+
+const WriteEmail = React.forwardRef<HTMLDivElement, WriteEmailProps>(
+  function DrawerMobileNavigation({ open, onClose }, ref) {
+    return (
+      <Sheet
+        ref={ref}
+        sx={{
+          alignItems: 'center',
+          px: 1.5,
+          py: 1.5,
+          ml: 'auto',
+          width: { xs: '100dvw', md: 600 },
+          flexGrow: 1,
+          border: '1px solid',
+          borderRadius: 'md',
+          backgroundColor: 'var(--joy-palette-background-level1)',
+          borderColor: 'var(--joy-palette-neutral-outlinedBorder)',
```

Comment:
```suggestion
          backgroundColor: 'background.level1',
          borderColor: 'neutral.outlinedBorder',
```
Better to use shorthand because the result will use the prefix from the theme.